### DISCUSSION TITLE
Zoom out in Android app.

### DISCUSSION THREAD

**Discussion by kimmilde** [home-assistant/frontend#10839] [Other feature requests]
I would like to have the possibility to zoom in or out. In ios you can chooose between pinch to zoom or fixed zoom (for instance 0.5-0.9x zoom). This function would have been great in Android app. Then I could compress more objects into one screen.

**Comment by conorlap** [comment]
Yes! This would be a great addition to the Android app, particularly for video streams!

**Comment by dshokouhi** [comment]
For feature requests in the android app we actually request an enhancement be filed in the android repo directly. This will need to be handled by our WebView Client. Pinch to zoom may not be possible but we can add a few options for users to switch between in the settings section most likely

The zooming in will not impact the video player, there is already a separate request for the video player functionality here: https://github.com/home-assistant/android/issues/1117
